### PR TITLE
feat(614): [2] add tokens to pipeline model

### PR DESCRIPTION
**Pull request by s-yoshika**
## Context
To implement pipeline API token.

## Objective
This PR allows pipeline models to contain their own tokens.

## References
Original issue:
https://github.com/screwdriver-cd/screwdriver/issues/614

Related PRs:
https://github.com/screwdriver-cd/data-schema/pull/257

### CODE REVIEW

**Review by tkyi — COMMENTED** [review]
Can you update the README with something like https://github.com/screwdriver-cd/models#tokens?

File: lib/pipeline.js [hunk 1]
```diff
@@ -676,6 +676,40 @@ class PipelineModel extends BaseModel {
...
+            }
+        };
+
+        // Lazy load factory dependency to prevent circular dependency issues
+        // https://nodejs.org/api/modules.html#modules_cycles
+        /* eslint-disable global-require */
+        const TokenFactory = require('./tokenFactory');
+        /* eslint-enable global-require */
+        const factory = TokenFactory.getInstance();
+        const tokens = factory.list(listConfig);
```
Comment (d2lam) [inline lib/pipeline.js:703]: Doesn't this return a promise?

Comment (d2lam) [reply lib/pipeline.js:703]: nvm, I see you just return below. Should be fine 👍

**Review by d2lam — APPROVED** [review]

**Comment by s-yoshika** [comment]
> Can you update the README with something like https://github.com/screwdriver-cd/models#tokens?

Thanks! I'll update it.

**Review by tkyi — APPROVED** [review]

**Pull request merged**